framework/include/mfem/solvers/MFEMSolverBase.h

```diff
@@ -23,7 +23,7 @@ class MFEMSolverBase : public MFEMGeneralUserObject
   /// Updates the solver with the given bilinear form and essential dof list, in case an LOR or algebraic solver is needed.
   virtual void updateSolver(mfem::ParBilinearForm & a, mfem::Array<int> & tdofs) = 0;
 
-  bool isLOR() const { return _lor; }
+  bool isLOR() const { return _lor || (_preconditioner ? _preconditioner->isLOR() : false); }
```

Comment:
```suggestion
  bool isLOR() const { return _lor || (_preconditioner && _preconditioner->isLOR()); 
```

Reply:
Some doxygen for why we need this new `||` would be great

Reply:
Oh that's embarrassing, good catch!